src/v0/destinations/tiktok_ads/util.test.js

```diff
@@ -95,3 +95,80 @@ describe('getContents utility test', () => {
     expect(expectedOutput).toEqual(getContents(message));
   });
 });
+
+describe('hashUserField utility test', () => {
+  it('should return the updated user object with hashed fields', () => {
+    const user = {
+      external_id: 123,
+      email: 'test@example.com',
+      phone: '+1234567890',
+    };
+
+    const hashedUser = hashUserField(user);
+
+    expect(hashedUser).toEqual({
+      external_id: expect.any(String),
```

Comment:
Why are we expecting any string? It should be sha256 encryption of 123 right?

Reply:
changed